### PR TITLE
Added new permission to docs

### PR DESCRIPTION
**Pull request by jasonf20**
Important to keep this page up-to-date. Otherwise customers might see something different when they integrate and they won't trust what it says.

### CODE REVIEW

**Comment by jasonf20** [comment]
Related to: https://github.com/Upsolver/TBD/commit/4b8ba2f5a553f28345d3fb8b945909da3d6792c2

File: Deployments/AWS/role-permissions.md [hunk 1]
```diff
@@ -37,7 +37,8 @@ Custom Policy:
 * `ec2:DescribeInstances`, `ec2:DescribeSpotInstanceRequests`, 
   `ec2:DescribeInstanceStatus`, `ec2:CreateTags`, `ec2:DescribeTags` - Allows monitoring Upsolver EC2 clusters
 * `ec2:DescribeSecurityGroups`, `ec2:DescribeImages`, 
-  `ec2:DescribeImageAttribute` - Required for Spotinst for validation
+  `ec2:DescribeImageAttribute`, - Required for Spotinst for validation
```
Comment (shanielh) [inline Deployments/AWS/role-permissions.md]:
```suggestion
  `ec2:DescribeImageAttribute` - Required for Spotinst for validation
```

File: Deployments/AWS/role-permissions.md [hunk 2]
```diff
@@ -37,7 +37,8 @@ Custom Policy:
...
+  `ec2:DescribeSpotPriceHistory`, - Required for Spotinst to use spot instances
```
Comment (shanielh) [inline Deployments/AWS/role-permissions.md]:
```suggestion
 * `ec2:DescribeSpotPriceHistory` - Required for Spotinst to use spot instances
```

File: Deployments/AWS/role-permissions.md [hunk 3]
```diff
@@ -73,6 +74,7 @@ Custom Policy:
                 "ec2:DescribeTags",
                 "ec2:DescribeImages",
                 "ec2:DescribeImageAttribute",
+		"ec2:DescribeSpotPriceHistory",
```
Comment (shanielh) [inline Deployments/AWS/role-permissions.md]:
```suggestion
	 	"ec2:DescribeSpotPriceHistory",
```

**Comment by jasonf20** [comment]
@shanielh Can you deploy this?

**Comment by shanielh** [comment]
@royh-upsolver please deploy the docs :)

**Comment by royh-upsolver** [comment]
@jasonf20 deployed - please verify all your content is there.

**Comment by jasonf20** [comment]
It's a little badly formatted (odd since It looked fine on github) but its there :+1:

**Pull request merged**